health/health.d/mysql.conf

```diff
@@ -136,20 +136,29 @@ component: MySQL
 
 # galera node state
 
- template: mysql_galera_cluster_state
+ template: mysql_galera_cluster_state_warn
        on: mysql.galera_cluster_state
     class: Errors
      type: Database
 component: MySQL
-     calc: $state
+     calc: $donor + $joined
     every: 10s
-     warn: $this == 2 OR $this == 3
-     crit: $this == 0 OR $this == 1 OR $this >= 5
+     warn: $this != nan AND $this != 0
     delay: up 30s down 5m multiplier 1.5 max 1h
-     info: galera node state \
-           (0: Undefined, 1: Joining, 2: Donor/Desynced, 3: Joined, 4: Synced, 5: Inconsistent)
+     info: galera node state is either Donor/Desynced or Joined.
        to: dba
 
+ template: mysql_galera_cluster_state_crit
+       on: mysql.galera_cluster_state
+    class: Errors
+     type: Database
+component: MySQL
+     calc: $undefined + $joining + $error
+    every: 10s
+     crit: $this != nan AND $this != 0
+    delay: up 30s down 5m multiplier 1.5 max 1h
+     info: galera node state is either Undefined or Joining or Error.
+       to: dba
```

Comment:
Hey Ilya!

We should have a way to skip maybe both `calc` and `lookup`, and use directly the dimensions directly in `crit` and `warn`...

Reply:
Is that a question or a statement?

Reply:
A thought :-)

Reply:
I hope you'd say a commitment 😄